### ISSUE TITLE
Why you set the initial lr as low as 1e-7 in trian.py?

### ISSUE THREAD

**Issue by minzhouGithub** [jfzhang95/pytorch-deeplab-xception#9]
why set the lr so low? Have you try other values? such as 0.007

**Comment by jfzhang95** [comment]
I try to train the model with different learning rates. Experiments show that a higher learning rate will cause loss value to be NAN.
